packages/core/demo/demo-data/line.ts

```diff
@@ -121,6 +121,23 @@ export const lineTimeSeriesOptions = {
 	curve: "curveMonotoneX"
 };
 
+export const lineTimeSeriesWithCustomStroke = {
+	...lineTimeSeriesOptions,
+	title: "Line (custom stroke, fill colors)",
+	getStrokeColor: (datasetLabel, label, value, datum, originalStrokeColor)=>{
+		console.log(`originalStrokeColor: ${originalStrokeColor}`);
+		return value > 60000 ? '#FF0000' : originalStrokeColor;
+	},
+	getFillColor: (datasetLabel, label, value, datum, originalFillColor)=>{
+		console.log(`originalFillColor: ${originalFillColor}`);
+		return value > 60000 ? '#FF0000' : originalFillColor;
+	},
+	getIsFilled: (datasetLabel, label, value, datum, originalIsFilled)=>{
+		console.log(`originalIsFilled: ${originalIsFilled}`);
+		return value > 60000 ? true : originalIsFilled;
+	}
```

Comment:
Seems like the 3 functions above need to go into fields that address the specific graphs.

e.g. what if you need different functions for line, then the scatter graphs, and the bars if you were using a combo chart?

Seems like this could be an addition to an interface somewhere and then read by each graph.